.cirrus.yml

```diff
@@ -6,7 +6,7 @@ task:
   only_if: $CIRRUS_API_CREATED == "true"
 
   container:
-    image: ponylang/ponyc-ci-x86-64-unknown-linux-gnu-builder:20191105
+    image: ponylang/ponyc-ci-x86-64-unknown-linux-gnu-builder:20191106
```

Comment:
is that date correct? i'm surprised for this PR that the image is that old.

Reply:
I've been working on this for far too long.

Reply:
i'm sorry. that sounds like a pained message from your end.